tests/unit/moduleapi/reply.tcl

```diff
@@ -37,10 +37,6 @@ start_server {tags {"modules"}} {
 #        assert_equal {0 1 2} [r rw.attribute 3]
 #    }
 
-    test {RM_ReplyWithPush: an set reply} {
-        assert_equal {0 1 2} [r rw.push 3]
-    }
```

Comment:
maybe i already commented on that subject..
let's test attributes, as soon as the other PR makes it possible.